src/cute_graphics.cpp

```diff
@@ -1832,7 +1851,6 @@ static SDL_GPUGraphicsPipeline* s_build_pipeline(CF_ShaderInternal* shader, CF_R
 	pip_info.depthStencilState.frontStencilState.compareOp = s_wrap(state->stencil.front.compare);
 	pip_info.depthStencilState.compareMask = state->stencil.read_mask;
 	pip_info.depthStencilState.writeMask = state->stencil.write_mask;
-	pip_info.depthStencilState.reference = state->stencil.reference;
```

Comment:
I think you need to call `SDL_SetGPUStencilReference` in `cf_apply_shader` to apply the stencil still.

Reply:
Done in https://github.com/RandyGaul/cute_framework/pull/215/commits/b1cb661f67d38969ceb87beab824f744c313e5b7